innovatrics/dot-web-samples
# updated to new core-backend (v5)

we updated from core-v3 to core-v5.
also, small improvements:
- documentation fixes
- development-workflow package-updates, command-invocation-fixes

how to test:
1. open README.md, see the list of all samples, try all of them
2. open the source code, make sure that every sample that uses the core-server goes to v5, not v3.
